Resources/Maps/nss_pillar.yml

```diff
@@ -120083,7 +120149,7 @@ entities:
     type: Transform
   - dockedWith: 9479
     type: Docking
-  - SecondsUntilStateChange: -38.142303
+  - SecondsUntilStateChange: -415.531

```

Comment:
Are these some kind of weird saving bug? What does -415 seconds do?

Reply:
Yeah got nothing to do with me. Tbh docking airlocks are bugged, they're permanently stuck to the station.